src/select/styles/select.m.css

```diff
@@ -12,6 +12,7 @@
 	background: var(--component-background);
 	border: 1px solid var(--component-color);
 	font-size: inherit;
+	min-height: 25px;
```

Comment:
do we have this in a variable somewhere for consistency ?

Reply:
Not really, since we aren't setting the heights of inputs in any standardized way. Perhaps we should go through the widgets styling and do a variable setup similar to the one we use in the dojo theme? I think it's outside the scope of this PR though.